specs/disk_details_for_hardware_inventory.adoc

```diff
@@ -0,0 +1,113 @@
+= Specification for adding disk details in node hardware inventory
```

Comment:
Please refer my comments from: https://github.com/Tendrl/specifications/pull/19#pullrequestreview-11780721 . More or less all the suggestions hold good here as well